app/views/apipie/apipies/_method_detail.erb

```diff
@@ -40,6 +40,8 @@
   <%= heading(t('apipie.returns'), h_level) %>
   <% method[:returns].each do |item| %>
     <%= heading("#{t('apipie.code')}: #{item[:code]}", h_level + 1) %>
+    <%= heading("#{t('apipie.description')}:", h_level + 2) %>
+    <p><%= item[:description] %></p>
```

Comment:
Shouldn't we check on the presence of the `item[:description]` before adding this?

Reply:
@X1ting ping?

Reply:
Ough, sorry, forgot to answer. Will change tomorrow. Thanks for review!